s2core/src/test/scala/com/kakao/s2graph/core/Integrate/QueryTest.scala

```diff
@@ -367,6 +368,40 @@ class QueryTest extends IntegrateCommon with BeforeAndAfterEach {
     edgesTo.reverse should be(ascOrderByTo)
   }
 
```

Comment:
another nice addition on our test cases. and I agree that changing insertEdgesSync into mutateEdgesSync is better.